modules/virt-adding-pci-device-hyperconverged-operator.adoc

```diff
@@ -25,23 +25,34 @@ metadata:
 spec:
   permittedHostDevices: <1>
     pciHostDevices: <2>
-      - pciVendorSelector: 10de:1eb8 <3>
-        resourceName: "nvidia.com/TU104GL_Tesla_T4" <4>
+    - pciDeviceSelector: "10DE:1DB6"  <3>
+      resourceName: "nvidia.com/GV100GL_Tesla_V100" <4>
+    - pciDeviceSelector: "10DE:1EB8"
+      resourceName: "nvidia.com/TU104GL_Tesla_T4"
+      disabled: true <5>
```

Comment:
not sure we need the `disable` flag in this example. Not sure we don't either...

Reply:
Removed the `disabled` flag from this example